server/commands.lua

```diff
@@ -288,8 +288,11 @@ QBCore.Commands.Add('me', Lang:t("command.me.help"), {{name = Lang:t("command.me
         local Player = Players[i]
         local target = GetPlayerPed(Player)
         local tCoords = GetEntityCoords(target)
+        local Player = QBCore.Functions.GetPlayer(source)
+        local citizenID = Player.PlayerData.citizenid
```

Comment:
Please move both `Player` and `citizenID` outside of the for loop, since we only need to get this data once.